android/app/src/main/java/com/google/codeu/chatme/presenter/CreateConversationPresenter.java

```diff
@@ -46,4 +46,23 @@ public void onComplete(DatabaseError databaseError, DatabaseReference databaseRe
                     }
                 });
     }
+
+    @Override
+    public void addGroupConversation(Conversation conversation) {
```

Comment:
Seems to be working after the changes. Please verify and let me if you had something else in mind.